src/apps/search/Place.tsx

```diff
@@ -132,6 +133,7 @@ const Place = () => {
         <LocationMarkers
           animate
           boundingBoxOptions={bboxOptions}
+          fitBoundingBox={Object.hasOwn(config.search, "zoom_to_place") ? config.search.zoom_to_place : true}
```

Comment:
The `underscore` library has a nice [get](https://underscorejs.org/#get) function that we could probably use here to avoid a ternary operator. Single quotes on string values.